### PR TITLE
WWST-1624 Add Sengled Element bulb with motion sensor

### PR DESCRIPTION
**Pull request by dkirker**

### CODE REVIEW

**Comment by dkirker** [comment]
@greens @varzac I moved the PR. I also removed the enrollResponse and enroll request handling. I also made a minor cosmetic change to move the checkInterval call into a helper function.

File: devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+}
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+	return zigbee.onOffRefresh() + zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS)
+}
+
+def refresh() {
+	state.enrollResponseCount = 0
```
Comment (varzac) [inline devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy]: I don't think this is used anymore.

**Review by varzac — DISMISSED** [review]

File: devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy (hunk 1 above)
Comment (varzac) [inline devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy]:
```suggestion
```

**Comment by varzac** [comment]
updated

**Review by varzac — APPROVED** [review]

File: devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy (hunk 1 above)
Comment (dkirker) [reply devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy]: d'oh! And wow! That's a cool new feature.

Comment (varzac) [reply devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy]: Yeah, and it looks like you would have been able to directly apply it through github (although i don't know what that would look like).  But apparently we were in a hurry so I just did what I knew.

Comment (dkirker) [reply devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy]: That's fine with me! There's a box below the suggestion, which is now greyed out, that says "Apply suggestion". That's pretty sweet.

**Pull request merged**